tawk_to.admin.inc

```diff
@@ -0,0 +1,260 @@
+                                                foreach ($whitelist as $page) { echo $page."\r\n"; }
+                                            }?></textarea>
+                                            <div class="description">
+                                                Add URLs to pages in which you would like to show the widget.<br>
+                                                Put each URL in a new line and just include the the leading '/' and page URL (e.g. /about)
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </fieldset>
+                            <fieldset class="form-wrapper" id="privacy-settings">
+                                <legend><span class="fieldset-legend">Privacy Settings</span></legend>
+                                <div class="fieldset-wrapper">
+                                    <div class="fieldset-description">Define whether or not user information can be used</div>
+                                    <div class="form-item form-type-checkbox">
+                                        <?php
+                                            $checked = 'checked';
+                                            if (!is_null($display_opts) && !$display_opts->enable_visitor_recognition) {
+                                                $checked = '';
+                                            }
+                                        ?>
+                                        <input type="checkbox" class="checkbox" name="enable_visitor_recognition" id="enable_visitor_recognition" value="1"
+                                            <?php echo $checked ?> />
+                                        <label class="option" for="enable_visitor_recognition">Enable visitor recognition</label>
+                                        <div class="description">
+                                            If selected, name and email address from logged in users will be used to identify the user to you when a chat comes in via tawk.to
+                                        </div>
+                                    </div>  
+                                </div>
+                            </fieldset>
+                            <div class="form-actions" id="edit-actions">
+                                <input class="button-primary form-submit" type="submit" id="module_form_submit_btn" name="submitBlockCategories" value="Save configuration">
+                            </div>                      
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <script type="text/javascript">
+        var currentHost = window.location.protocol + "//" + window.location.host;
+        var url = "<?php echo $iframe_url; ?>&pltf=drupal&pltfv=7&parentDomain=" + currentHost;
```

Comment:
With regards to this issue https://github.com/tawk/tawk-backdrop/issues/2, you can go ahead and change the `pltf=drupal` to `pltf=backdrop` and remove `pltfv=7`

Reply:
Done